decidim-proposals/app/controllers/decidim/proposals/proposals_controller.rb

```diff
@@ -90,6 +90,22 @@ def update
         end
       end
 
+      def withdraw
+        @proposal = Proposal.not_hidden.where(feature: current_feature).find(params[:id])
+        authorize! :withdraw, @proposal
+
+        WithdrawProposal.call(@proposal, current_user) do
```

Comment:
It looks like the lines below (at least the :ok scenario) aren't being tested. Maybe we should add an acceptance test for this?

Reply:
It is not tested because of this issue here: https://github.com/decidim/decidim/issues/2471 Redirects do not work well, so we cannot test it, and I don't have any idea on how to solve this 😞

Reply:
Oh my :(